akka-distributed-data/src/test/scala/akka/cluster/ddata/WriteAggregatorSpec.scala

```diff
@@ -181,6 +189,82 @@ class WriteAggregatorSpec extends AkkaSpec(s"""
     }
   }
 
+  "WriteAggregator with delta" must {
+    implicit val cluster = Cluster(system)
+    val fullState1 = ORSet.empty[String] + "a" + "b"
+    val fullState2 = fullState1.resetDelta + "c"
+    val delta = Delta(DataEnvelope(fullState2.delta.get), 2L, 2L)
+
+    "send deltas first" in {
+      val probe = TestProbe()
+      val aggr = system.actorOf(WriteAggregatorSpec.writeAggregatorPropsWithDelta(
+        fullState2, delta, writeMajority, probes(probe.ref), nodes, Set.empty, testActor, durable = false))
+
+      probe.expectMsgType[DeltaPropagation]
+      probe.lastSender ! WriteAck
+      probe.expectMsgType[DeltaPropagation]
+      probe.lastSender ! WriteAck
```

Comment:
I think this test only verifies that it tries with `DeltaPropagation` messages first.
`val nodes = Set(nodeA, nodeB, nodeC, nodeD)`
so it should send out a few such that are delivered to the `probe`